chainer/functions/math/det.py

```diff
@@ -0,0 +1,106 @@
+import numpy.linalg
+
+from chainer import cuda
+from chainer import function
+from chainer.functions.array.reshape import Reshape
+from chainer.functions.math.inv import _inv_gpu
+from chainer.functions.math.matmul import _as_batch_mat
+from chainer.functions.math.matmul import _check_ndim
+from chainer.functions.math.matmul import _convert_type
+from chainer.functions.math.matmul import _get_ld
+from chainer.functions.math.matmul import _mat_ptrs
```

Comment:
According to pep8, objects which begin with single underscore are for private use (see e.g. [here] http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python). So consider make these functions public.